docs/Contributing.md

```diff
@@ -0,0 +1,84 @@
+# Contributing
+
+This document is designed to act as an outline for the process of contributing to the project. You will find some workflow tips, naming conventions, and other useful information. A standard process will increase productivity.
+
+## Outline
+
+As you begin adding code to the repository you will notice there is a lot of code. And we have a standard naming convention for folders and files that you should follow.
+
+1. Fork the repository, and keep your `master` branch the same as `messari/subgraphs`, before creating a feature branch in your repository **pull** upstream changes.
+2. Create your feature branch `branch-name`.
+3. Once you begin to **push** changes create a Pull Request (PR) in `messari/subgraphs` and set it as a draft until you it is ready for review.
+
+![How to convert to draft](./images/contributing/convert-to-draft.png)
+
+> See "Naming Conventions" for how to name your PR
+
+4. Once your PR is ready for review go ahead and click "Ready for review". As a courtesy it is nice to let the reviewer know that it is ready through a DM.
+
+At this point you will just need to follow the "Reviewing" process until your PR is merged.
+
+## Reviewing
+
+### Reviewee
```

Comment:
How should the reviewer be chosen?

Reply:
made a comment, right now there isn't a standard. usually vincent assigns it and it's someone who has previously worked on the code or something similar